### DISCUSSION TITLE
After Upgrading to 3.4 I can't open PDFs from my mail (Forte Agent)

### DISCUSSION THREAD

**Discussion by LarsErikOsterud** [sumatrapdfreader/sumatrapdf#3626] [General]
Just get either "There is no application assiciated with application/pdf" (Windows 10 machine)
or "Unable to execute command "c:\temp\<filename>.pdf" (Windows 11).
Why doesn't Forte Agent recognize SumatraPDF as the app for PDF after upgrading from 3.2 to 3.4?
Or has this got something to do with 32 vs 64 bits versions or something?  Help?

**Comment by GitHubRulesOK** [comment]
Your problem is really outside of SumatraPDF remit
HOWEVER MS tightened PDF security when they introduced Edge such that Edge takes over PDF FTA control from the user by using  a security Hash mechanism and often claws back File Type assignment control 
That should be less  likely now as many user probably complained, but the legacy hash method lingers. You should be able to follow the recommended solution in docs.
https://www.sumatrapdfreader.org/docs/Set-as-default-PDF-viewer

But for any app like Forte you need to check with their support channels.

**Comment by LarsErikOsterud** [comment]
But this worked file intil suddenly today (I got an update for both Win10 and WIn11).
It seems like mye mail program is not allow to open SumatraPDF suddenly to launch the attachment.
On Win10 I can brwose for an app an d select SumatraPDF. Then it opens. But it does not remember to next time.

Seems to be something even stranger. If I click a PDF download link in my browser (Firefox) usually the file is downloade dand opened in SumatrPDF, but now it just opens inside Firefox.  Just like it ignores that SumatraPDF is aassciated with the PDF type.

**Reply by GitHubRulesOK** [reply]
that is what Edge did (still does?) it claws back control on any change to the hash
so you need to keep slapping it with "Its my computer , so reset PDF to my preference I only paid MS for the OS"
follow the guide to **attempt** to tell Windows that Edge is not King of the PDF Domain.
